ibazel/main.go

```diff
@@ -94,27 +94,30 @@ func main() {
 		return
 	}
 
+	// Create a timer objet to use for debouncing. When the timer is created, do
+	// not start the timer but instead start the timer when an event that builds
+	// should be run.  This has the added benefit of kicking things off by
```

Comment:
not sure what you mean by "start the timer" here... NewTimer will start the timer upon creation.